### PR TITLE
fix(gen): fixed 301s

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx [hunk 1]
```diff
@@ -152,6 +152,6 @@ Besides creating your Elastic Metal servers from the graphical [Scaleway console
 
     Your new Elastic Metal server will be installed and ready to be used in usually less than **15 minutes**, all API features are listed in the [API documentation](https://www.scaleway.com/en/developers/api/elastic-metal/).
 
-For more information about Elastic Metal servers, refer to our [dedicated product documentation](/bare-metal/elastic-metal/) and the [FAQ](https://www.scaleway.com/en/faq/elastic-metal-servers/).
+For more information about Elastic Metal servers, refer to our [dedicated product documentation](/bare-metal/elastic-metal/) and the [FAQ](https://www.scaleway.com/en/docs/faq/elastic-metal/).
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx:155]: Why is the link here full and not the usual `(docs/faq/elastic-metal/)`?

Comment (nerda-codes) [inline bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx:155]:
```suggestion
For more information about Elastic Metal servers, refer to our [dedicated product documentation](/bare-metal/elastic-metal/) and the [FAQ](/faq/elastic-metal/).
```

File: compute/instances/index.mdx [hunk 2]
```diff
@@ -45,6 +45,8 @@ meta:
     />
 </Grid>
 
+[This is a test](https://www.scaleway.com/en/docs/console/my-project/how-to/join-an-organization/)
```
Comment (RoRoJ) [inline compute/instances/index.mdx:48]:
```suggestion
```

File: compute/instances/index.mdx [hunk 3]
```diff
@@ -45,6 +45,8 @@ meta:
...
+
```
Comment (nerda-codes) [inline compute/instances/index.mdx:49]: Is the link a test too?

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**